packages/sit-onyx/src/components/OnyxRipple/OnyxRipple.ct.tsx

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from "../../playwright/a11y";
+import OnyxRipple from "./OnyxRipple.vue";
+
+test("should trigger some ripples", async ({ mount }) => {
+  // ARRANGE
+  const component = await mount(
+    <OnyxRipple enter-duration="2000ms" style="width: 500px; height: 500px;" />,
```

Comment:
```suggestion
    <OnyxRipple style="width: 500px; height: 500px;" />,
```

The prop no longer exists, right?